docs/make.jl

```diff
@@ -16,6 +16,9 @@ readme = replace(readme, r"<img src=\"([^\"]+)\"[^>]+>.*" => s"![](\1)")
 # Then, we remove any line with "<div" on it:
 readme = replace(readme, r"<[/]?div.*" => s"")
 
+# Then, we titlecase the ToC entries to be compatible with Documenter.jl
+readme = replace(readme, r"\[(.*?)\]\(#(.*?)\)" => titlecase)
```

Comment:
Where is `titlecase` defined? Is it some function? If so, maybe do an explicit import. I think for less-common imports it's very helpful for readability to have explicit imports

Reply:
For sure, it's exported in Base. Added an explicit import 2b397a6

Reply:
Oh, sorry, I had no idea this was in Base... my bad. You can remove the import then